FlowersStore/Scenes/FlowersList/FlowersListPresenter.swift

```diff
@@ -20,16 +20,26 @@ final class FlowersListPresenter {
     private weak var output: FlowersListPresenterOutput?
     private let interactor: FlowersListInteractorInput
     
+    private var teste: String?
+    
     init(interactor: FlowersListInteractorInput = FlowersListInteractor()) {
         self.interactor = interactor
     }
     
     func setOutput(_ output: FlowersListPresenterOutput?) {
```

Comment:
<!--
  0 failure: 
  1 warning:  Function 'setOutp...
  
  
  DangerID: danger-id-Danger;
  File: FlowersStore/Scenes/FlowersList/FlowersListPresenter.swift;
  Line: 29;
-->

- :warning: Function 'setOutput(_:)' is unused